councilmatic_core/management/commands/import_data.py

```diff
@@ -981,6 +983,10 @@ def insert_raw_bills(self, delete=False):
 
             slug = slugify(bill_info['identifier'])
 
+            restrict_view = False
+            if bill_info['extras'].get('restrict_view'):
+                restrict_view = bill_info['extras']['restrict_view']
```

Comment:
This is going to be an actual Boolean value here? Or is it something like yes/no t/f, etc? I guess that's something that would ultimately need to be fixed in the scrapers but for a boolean field we should probably make sure it's consistent.

Reply:
yep - it's a boolean. Here's a bill with that field, as an example: https://ocd.datamade.us/ocd-bill/d4feee2c-56a5-427d-b8d7-f50603055a13/

Reply:
Great! And, just to be super paranoid, LA Metro is using the Granicus API, right? So hypothetically that data type should be more consistent?

Reply:
yep! you can see an example of that field in the API: http://webapi.legistar.com/v1/metro/matters/5261

from my understanding, this field gets toggled to "false" (i.e., make it viewable) when the Metro staff clicks a checkbox. So, there should not be any variance in the field, e.g., Metro staff could not set it to "t" or "f" or "kitten"